pymodbus/transport/transport.py

```diff
@@ -397,11 +399,11 @@ def transport_send(self, data: bytes, addr: tuple = None) -> None:
             self.sent_buffer += data
         if self.comm_params.comm_type == CommType.UDP:
             if addr:
-                self.transport.sendto(data, addr=addr)  # type: ignore[attr-defined]
+                self.transport.sendto(data, addr=addr)  # type: ignore[union-attr]
```

Comment:
This looks like something that can be avoided !

Reply:
Not if `transport = None`.

Perhaps a better answer here is some sort of dummy transport instead of `None`

Reply:
That is a variant, which is acceptable.

Reply:
I tried the dummy transport; it didn't work well.